### ISSUE TITLE
Logic behind "GREEN", "YELLOW", "RED" status

### ISSUE THREAD

**Issue by epsi95** [nic-delhi/AarogyaSetu_Android#466]
I am trying to know the logic behind the status with color "RED", "GREEN", "RED" that is shown on dashboard. I looked into the Dashboard Activity but it seems that it is using webview so I assume the whole dashboard is created at server side and sent it to mobile.
1. Does that mean the color logic is implemented at server side?
1.a.  If that is correct, I want to know the logic that is determining the color and the parameters that is taken as input. A code snippet will be helpful.

2. If the above is not correct than it means the app is doing some conditional check while taking input from user. (Forgive me that I didn't read the whole source code, I just want to know the logic). If that is the case, kindly tell me the class name into which I should look into.

So in a nutshell, I want to know 
1. Parameters taken as input (user q/a inside app, ...etc?)
2. Conditions that is determining     the color as "RED", "GREEN", "YELLOW"
